implementations/java/org.hl7.fhir.r4/src/org/hl7/fhir/r4/test/support/TestingUtilities.java

```diff
@@ -214,21 +214,26 @@ public static String checkJsonSrcIsSame(String s1, String s2) throws JsonSyntaxE
   public static String checkJsonSrcIsSame(String s1, String s2, boolean showDiff) throws JsonSyntaxException, FileNotFoundException, IOException {
     String result = compareJsonSrc(s1, s2);
     if (result != null && SHOW_DIFF && showDiff) {
-      
       String diff = null; 
-      if (System.getProperty("os.name").contains("win"))
-        diff = Utilities.path("usr", "bin", "meld");
-      else
-        diff = Utilities.path(System.getenv("ProgramFiles(X86)"), "WinMerge", "WinMergeU.exe");
+      if (System.getProperty("os.name").contains("Linux"))
+        diff = Utilities.path("/", "usr", "bin", "meld");
+    	else {
+    		if (Utilities.checkFile("WinMerge", Utilities.path(System.getenv("ProgramFiles(X86)"), "WinMerge"), "\WinMergeU.exe", null))
+    			diff = Utilities.path(System.getenv("ProgramFiles(X86)"), "WinMerge", "WinMergeU.exe");
+    		else

```

Comment:
Can you do an `else if` for the `else` part (Meld) so it skips the whole trying to open up a diff-tool if it can't find it.